README.md

```diff
@@ -70,6 +70,12 @@ the `[global]` section. This is useful when you run applications related to
 Values can be surrounded by double quotes (") which is useful if the value contains
 the comment character (#).
 
+The line "cache_path = cache_directory" defines where Spotify's cache-files are stored. 
+These files are used to avoid downloading a file again when a track is replayed. 
+Here it's set to "cache_directory" and the cache files will be stored in "/cache_directory".
```

Comment:
I think
Here it's => Here its

https://grammarist.com/spelling/its-its/

Reply:
> and the cache files will be stored in "/cache_directory".

Might be worth mentioning that:
 - tilde-expansion `~/cache/spotifyd` won't work
 - variable `${HOME}/cache/spotifyd` won't work
 - the directory needs to exist

Reply:
> I think
> Here it's => Here its
> 
> https://grammarist.com/spelling/its-its/

No, it is good as it is.